src/extractor/extraction_containers.cpp

```diff
@@ -323,8 +323,8 @@ void ExtractionContainers::PrepareEdges(ScriptingEnvironment &scripting_environm
     {
         if (edge_iterator->result.osm_source_id < node_iterator->node_id)
         {
-            util::SimpleLogger().Write(LogLevel::logWARNING) << "Found invalid node reference "
-                                                             << edge_iterator->result.source;
+            util::SimpleLogger().Write(LogLevel::logDEBUG) << "Found invalid node reference "
```

Comment:
Hrm this one is kind of important - if a node id is not found the graph will potentially be corrupted. We had this case once for Highway 1 in CA breaking our routing in California.

So I would not switch it to debug here.